README.md

```diff
@@ -93,21 +100,120 @@ Collecting logs
 
 To record the logs from your containers specify a location:
 
-    public class DockerCompositionTest {
-  
-        @ClassRule
-        public DockerComposition composition = new DockerComposition("src/test/resources/docker-compose.yml")
-                                                      .saveLogsTo("build/dockerLogs/dockerCompositionTest");
-  
-        @Test
-        public void testRecordsLogs() throws InterruptedException, IOException {
-           ...
-        }
-  
+```java
+public class DockerCompositionTest {
+
+    @ClassRule
+    public DockerComposition composition = new DockerComposition("src/test/resources/docker-compose.yml")
+                                                  .saveLogsTo("build/dockerLogs/dockerCompositionTest");
+
+    @Test
+    public void testRecordsLogs() throws InterruptedException, IOException {
+       ...
     }
 
+}
+```
+
 This will automatically record logs for all containers in real time to the specified directory. Collection will stop when the containers terminate.
 
+Docker Machine
+--------------
+
+Docker is able to connect to daemon's that either live on the machine where the client is running, or somewhere remote.
+Using the `docker` client, you are able to control which daemon to connect to using the `DOCKER_HOST` environment
+variable.
+
+We introduce the concept of a Local Machine and a Remote Machine, to make the choice about which Docker daemon you're
```

Comment:
I'd write the docs in a declarative style - stating what is rather than what we're changing.